carsCollection.php

```diff
@@ -19,7 +19,7 @@
 <body>
 <section>
     <div class="button">
-        <a class="addmore" href="./index.php">Add more cars</a>
+        <a class="addmore" href="storyTwo.php">Add more cars</a>
```

Comment:
Could you rename this to describe the code that is in the file? Something like `form.php` or `addCar.php`